### PR TITLE
fix: fixing styles for fixed positioned filters

### PR DESCRIPTION
**Pull request by diego-antonelli**
- Add styles to fix the wrong positioning when virtual scrolling is on and there is no rows in the DG.

### CODE REVIEW

File: themesource/datagrid/web/datagrid.scss [hunk 1]
```diff
@@ -255,3 +254,32 @@ $brand-primary: #264ae5 !default;
...
+        align-items: center;
+        cursor: pointer;
+
+        label {
+            margin: 8px;
+            font-weight: normal;
+            white-space: nowrap;
+        }
+    }
+}
```
Comment (IIsaku) [inline themesource/datagrid/web/datagrid.scss]: Please add a newline here.

File: themesource/datagrid/web/datagrid-filters.scss [hunk 2]
```diff
@@ -94,6 +94,81 @@ $arrow: "data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='
     }
 }
 
+.filter-selectors {
```
Comment (IIsaku) [inline themesource/datagrid/web/datagrid-filters.scss:97]: I noticed the outline styles being cut off at the rounded corners of the filter menu. Is that issue introduced with these changes? Or was that already the case?

Comment (diego-antonelli) [reply themesource/datagrid/web/datagrid-filters.scss:97]: I will leave this one for future improvement as it only happens in Chrome

File: themesource/datagrid/web/datagrid.scss (hunk 1 above)
Comment (diego-antonelli) [reply themesource/datagrid/web/datagrid.scss]: Fixed

**Review by IIsaku — APPROVED** [review]

**Review by tamelbrun — APPROVED** [review]

**Pull request merged**